peep.sh

```diff
@@ -8,27 +8,17 @@ ARGS=""
 
 echo "peep.sh: Using pip $PIPVER"
 
-# If we're using pip 1.5, 1.6 or 6.0, then toss on the --no-use-wheel
-# argument.
-#
-# Note: If we encounter other versions of pip here that require other
-# things, we can toss them in.
```

Comment:
I'd keep this comment in. It's helpful to future people who might fix this file.